user_guide_src/source/libraries/uploaded_files.rst

```diff
@@ -126,6 +126,13 @@ In controller::
 	$file1 = $this->request->getFile('images.0');
 	$file2 = $this->request->getFile('images.1');
 
+You can access individual files uploaded at the same time, but with different names,
+using ``getFileMultiple()``::
+
+	$file1 = $this->request->getFileMultiple('somefile');
+	$file2 = $this->request->getFileMultiple('adifferentfile');
+
```

Comment:
This is wrong, because `getFileMultiple()` return an array with the files, so should be something like this:

`$files = $this->request->getFileMultiple('somefile');`
`$file1 = $files[0];`
`$file2 = $files[1];`

Reply:
Agreed. I missed that earlier, and should have had more patience and waited for the user guide updates with your PR :(